pwa/package.json

```diff
@@ -34,6 +36,7 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "oazapfts": "^6.1.0",
```

Comment:
I haven't used this in the past, but I've used `openapi-typescript`. Did you look into the differences between the two? I don't have strong opinions either way but `openapi-typescript` looks to be used more.

Reply:
Yeah I tried out both. `openapi-typescript` is definitely more popular BUT for our specific API I think it's much more clunky:

```ts
const { data, error } = await client.GET("/team/{team_key}", {
  params: {
    path: { team_key: "frc604" },
  },
});
```

It's _really_ verbose. `oazapfts` is simply `await client.getTeam("frc604");`

Let me do one more round of digging into this to make sure I haven't missed anything with `openapi-typescript`.

Reply:
Took another look and not seeing a way to do it with `openapi-typescript` that makes logical sense and is easy to use. So I think oazapfts is the way to go.

Reply:
Fair enough. Can you fix the package.json conflicts then i can merge